Ascii_Subsequences/ASCII_Subsequences.c

```diff
@@ -0,0 +1,114 @@
+/*ASCII SUBSEQUENCES PROBLEM
+This problem is same as the "Subsequences" question just instead of two function calls, three function calls will be made :
+- One with the same string.
+- One with a letter added.
+- One with the ASCII code of the letter added to the string.
+- For example:-
+Take as input str, a string. We are concerned with all the possible ASCII-subsequences of str. E.g. “ab” has following ASCII-subsequences “”, “b”, “98”, “a”, “ab”, “a98”, “97”, “97b”, “9798”
+a. Write a recursive function which returns the count of ASCII-subsequences for a given string. Print the value returned.
+b. Write a recursive function which prints all possible ASCII-subsequences for a given string (void is the return type for function).

```

Comment:
one line should contain max 80 characters. please change the comment according to it
a brief introduction is enough